packages/web-components/src/components/ic-pagination/ic-pagination.tsx

```diff
@@ -95,6 +100,12 @@ export class Pagination {
       return;
     }
 
+    this.startEllipsis = false;
+    this.endEllipsis = false;
+    this.startItems = [];
+    this.endItems = [];
+    this.midItems = [];
+
```

Comment:
Are the scoped variables below required if they are reset here (where relevant)?